### ISSUE TITLE
[BUG] localCIDER reports kappa value significantly higher than 1

### ISSUE THREAD

**Issue by MartinGirard** [Pappulab/localCIDER#17]
The sequence 'EDDLIKPEEEEEEEEEEEEEEEE' reports a value of kappa larger than 1.

To reproduce:
```
>>> import localcider
>>> sequence = 'EDDLIKPEEEEEEEEEEEEEEEE'
>>> print(localcider.sequenceParameters(sequence=sequence).get_kappa()
1.288434048040376
```

**Comment by alexholehouse** [comment]
Yes - this will happen sometimes (especially for short sequences with extreme compositions) because they fall into an edge case with the heuristics used. I would say for a sequence like this (with a single positively charged residue and 19 negatively charged residues), kappa is probably [definitely] not a relevant parameter.
